Arelle/Arelle
# Build installable mac distributions

#### Reason for change
To improve the release for arelle, we would like to build stable distributions as part of the CI instead of manually

#### Description of change
Builds installable distributions containing all necessary files to run arelle on any of the following macOsX versions for the following variants of arelle
- OSX 11
- OSX 12

#### Steps to Test
1. Create an os 11 and os 12 VM following the instructions in the `vms/mac/README.md`
2. From the each VM download the appropriate version of the mac build and drag into applications
3. from your terminal run `xattr -dr com.apple.quarantine /Applications/Arelle.app` (This bypasses mac security)
4. Visit http://filingindex.arelle.org/filings and download a 10Q.
    i. Search a company
    ii. Click on the 10Q button
    iii. Click on filing
    iv. On the sec website change the url from `-index.htm` to `-xbrl.zip` to download a filing zip
    v. If mac unzips the filing automatically navigate into the folder and rezip it
5. Enable the edgar renderer and ixbrl viewer
    i. Help manage plugins 
6. Disable cert checks
    i. tools/internet/disable cert checks 
5. Load the filing in arelle
    i. File load filing (navigate to where you saved your filing)
    ii. Verify that the filing is loaded
    iii. View the ixbrl viewer and the edgar renderer

**review**:
@Arelle/arelle
